tox.ini

```diff
@@ -11,12 +11,15 @@ extras =
     testing
     coverage: coverage
 deps =
+    setuptools<57
```

Comment:
This requires a comment explaining the motivation

Reply:
added a comment